src/molecules/product-list-item/ProductListItem.js

```diff
@@ -0,0 +1,95 @@
+// @vue/component
+export default {
+  model: {
+    prop: 'qty',
+    event: 'change'
+  },
+  props: {
+    /**
+     * Product id
+     */
+    id: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product name
+     */
+    name: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product url
+     */
+    url: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product image url
+     */
+    imageUrl: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product standard price
+     */
+    price: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product special price
+     */
+    specialPrice: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product old price
+     */
+    oldPrice: {
+      type: String,
+      required: true
+    },
+    /**
+     * Product options/attributes array (size, color etc)
+     */
+    options: {
+      type: Array,
+      default: null
+    },
+    /**
+     * Product quantity
+     */
+    qty: {
+      type: Number,
+      default: 1
+    },
```

Comment:
if we remove qty from default structure we can also remove this `qty` and `hideQty` props